TF migration docs - round 3 (#2038)
saucelabs/sauce-docs: docs/test-fairy/sdk/options.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

TestFairy requires that you call `begin` to start recording your sessions. However, developers can override the build settings to determine what is enabled during a session recording.

Some commonly used options:

- Crash Reporting
- Video Recording
- Recorded Metrics
- Max Session Length
- Feedback Form

<Tabs
groupId="platform"
defaultValue="android"
values={[
{label: 'Android', value: 'android'},
{label: 'iOS', value: 'ios'},
]}>

<TabItem value="android">

### Crash Reporting

TestFairy provides a means of capturing and recording stack traces if your application crashes. Stack traces can be vital to understanding any underlying bugs in your app. However, some apps may want to disable TestFairy's crash handling. Invoke `enableCrashHandler` or `disableCrashHandler` before calling `begin`.

> Once the TestFairy crash handler has been enabled, it cannot be disabled unless the app is restarted.

Author: ```suggestion
Once you enable the TestFairy crash handler, it cannot be disabled unless the app is restarted.
```

#### Syntax

```java
TestFairy.disableFeedbackForm();
TestFairy.enableFeedbackForm("<method>");
```

Refer to the [Class Reference](https://app.testfairy.com/reference/android/index.html) for more information on values for `method`.

#### Code Example

In the following example, feedback will be enabled when the device is shaken.

```java
import com.testfairy.TestFairy;

public class MyApplication extends Application {
 @Override
 public void onCreate() {
 super.onCreate();

 TestFairy.enableFeedbackForm("shake");
 TestFairy.begin(this, "<app token>");
 }
}
```

Once logged in, your app token is available from your [account preferences](https://app.testfairy.com/settings#apptoken).

</TabItem>

<TabItem value="ios">

### Crash Reporting

TestFairy provides a means of capturing and recording stack traces if your application crashes. Stack traces can be vital to understanding any underlying bugs in your app. However, some apps may want to disable TestFairy's crash handling. Invoke `enableCrashHandler` or `disableCrashHandler` before calling `begin`.

> Once the TestFairy crash handler has been enabled, it cannot be disabled unless the app is restarted.

Author: ```suggestion
Once you enable the TestFairy crash handler, it cannot be disabled unless the app is restarted.
```
